autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -59,13 +59,33 @@ def load(output_directory, verbosity=2):
 
     @staticmethod
     @unpack(set_presets)
-    def fit(train_data, label, tuning_data=None, time_limits=None, output_directory=None, presets=None,
-            problem_type=None, eval_metric=None, stopping_metric=None,
-            auto_stack=False, hyperparameter_tune=False, feature_prune=False,
-            holdout_frac=None, num_bagging_folds=0, num_bagging_sets=None, stack_ensemble_levels=0,
-            hyperparameters=None, num_trials=None, search_strategy='random', search_options=None,
-            nthreads_per_trial=None, ngpus_per_trial=None, dist_ip_addrs=None, visualizer='none',
-            verbosity=2, **kwargs):
+    def fit(train_data,
+            label,
+            tuning_data=None,
+            time_limits=None,
+            output_directory=None,
+            presets=None,
+            problem_type=None,
+            eval_metric=None,
+            stopping_metric=None,
+            auto_stack=False,
+            hyperparameter_tune=False,
+            feature_prune=False,
+            holdout_frac=None,
+            num_bagging_folds=0,
+            num_bagging_sets=None,
+            stack_ensemble_levels=0,
+            hyperparameters=None,
+            num_trials=None,
+            search_strategy='random',
+            search_options=None,
+            nthreads_per_trial=None,
+            ngpus_per_trial=None,
+            dist_ip_addrs=None,
+            visualizer='none',
+            verbosity=2,
+            scheduler_options=None,
```

Comment:
prefer to move scheduler_options to be directly before search_strategy so it is near similar arguments.

Reply:
OK